src/main/resources/resources/cohortdefinition/sql/locationRegion.sql

```diff
@@ -0,0 +1,21 @@
+-- Begin Location region Criteria
+select
+  C.entity_id AS person_id,
+  C.location_history_id as event_id,
+  C.start_date as start_date,
+  COALESCE(C.end_date, CAST('2099-12-31' AS DATE)) as end_date,
+  C.region_concept_id as TARGET_CONCEPT_ID,
+  CAST(NULL as bigint) as visit_occurrence_id
+from 
+(
+  select l.*, lh.*
+  FROM @cdm_database_schema.LOCATION l
+    JOIN @cdm_database_schema.LOCATION_HISTORY lh ON l.location_id = lh.location_id
+@codesetClause
+  WHERE lh.domain_id = (
+    SELECT CAST(concept_id AS VARCHAR)
```

Comment:
why do we need to cast a bigint field to varchar?

Reply:
@chrisknoll ,
![image](https://user-images.githubusercontent.com/10010071/55879845-873d1180-5b6d-11e9-9765-c04433c42f90.png)

Reply:
So, the concept_id in the query stores a value of either 'PERSON', 'PROVIDER' or 'CARE_SITE'? Those don't look like concept_ids.